### ISSUE TITLE
Train/Val/Test Split

### ISSUE THREAD

**Issue by egeozsoy** [Jingkang50/OpenPSG#106]
Hi, what is the currently recommended train/val/test split?

The line ```assert split in {'train', 'test'}``` confuses me a bit, as it indicates that there is no validation split?

**Comment by Jingkang50** [comment]
Actually, in the main paper, we used all the training set for training and test on the test set.
However, in the PSG competition that we held last year, we use `psg_train_val.json` (45697 training data + 1000 validation data with GT) as training and val splits, so that the rest are test set. But again, we dont have validation set in the original paper.
